src/native/libs/System.Security.Cryptography.Native.Apple/pal_ssl.c

```diff
@@ -641,8 +641,17 @@ int32_t AppleCryptoNative_SslSetEnabledCipherSuites(SSLContextRef sslContext, co
     }
 }
 
+PALEXPORT int32_t AppleCryptoNative_SslSetCertificateAuthorities(SSLContextRef sslContext, CFArrayRef certificates, int32_t replaceExisting)
+{
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+    // The underlying call handles NULL inputs, so just pass it through
+    return SSLSetCertificateAuthorities(sslContext, certificates, replaceExisting);
```

Comment:
This may be available only for macOS 10.15+. I'm not sure if we need to worry about anything older. And it seems to be missing on mobile platforms (or we target something older)

Reply:
I added indirection similar to `SSLSetALPNProtocols`